### PR TITLE
Bani

### PR DESCRIPTION
**Pull request by Bani2791**
## **1. Setup > Settings > Servers**

**Incorporated VALE suggestions**

https://bani--connexcs-docs.netlify.app/setup/settings/servers/

## **2. Customer > Ingress Routing**

**Locks> needs to add information about DNC List & Block Destination Type**

**DNC List**: It is a **Do Not Call** list. The customer will not be able to able to dial the numbers in the specified DNC list.
You can add the lsit of numbers in the Database.

**Block Destination Type**: You can select and block the calls to various destinations (carriers) like Mobile, Fixed, Paging etc.

**Spam Scout Scoring**: It blocks the Spam calls based on the CLIs.
You can either Block All, Allow All, Block Most Spam, or Block Least Spam

https://bani--connexcs-docs.netlify.app/customer/routing/#locks

## **3. Customer > Create Customer**

**3.1 Rewrite Portal Access > Config**

**Re-written statement**: **Portal Access:** It allows you to **Show** and **Hide** parameters like Balance, CDR, Breakout etc on your customer portal.

https://bani--connexcs-docs.netlify.app/customer/customer/

## **4. Logging**

**4.1 Add LRN to Call details**

https://bani--connexcs-docs.netlify.app/logging/

**4.2 Tidyup**

https://bani--connexcs-docs.netlify.app/logging/

## **5. Setup > Integrations > Portal**

**5.1 Changed the name **User Access** to **Configuring Users****

https://bani--connexcs-docs.netlify.app/setup/integrations/portal/#configuring-users

**5.2 Tidyup**

https://bani--connexcs-docs.netlify.app/setup/integrations/portal/

## **6. Setup > AnyEdge**

**Added the Pricing note:** !!! note "The monthly fee for the Anyedge service is $20."

https://bani--connexcs-docs.netlify.app/anyedge/anyedge/

### CODE REVIEW

File: docs/setup/settings/servers.md [hunk 1]
```diff
@@ -168,34 +169,34 @@ This only needs to be run when the system indicates "There is a pending update o
...
 
-Once a user is registered via UDP, the connection is verified once again.
+Once a user registers via UDP, the connection is verified once again.
 
 ### Deployment Options
 
 **1. Single Server**: No additional configuration is needed for a single server.
-**2. Server Array**: For multiple servers, decide whether users registered to *Server A* can call users on *Server B*. If so, set the **UAC Location Array Sharing** option (found under [**Server Config**](https://docs.connexcs.com/setup/settings/servers/#server-config)) for the servers which will make the registrations.
-**3. Server Cluster**: Servers in a cluster share all user location registrations. This is the best solution for scaling to many thousands of registrations.
+**2. Server Array**: For various servers, decide whether users registered to *Server A* can call users on *Server B*. If so, set the **UAC Location Array Sharing** option (found under [**Server Config**](https://docs.connexcs.com/setup/settings/servers/#server-config)) for the servers which will make the registrations.
```
Comment (digipigeon) [inline docs/setup/settings/servers.md:193]: Various is not a suitable replacement for multiple in this sentence.

File: docs/setup/settings/servers.md [hunk 2]
```diff
@@ -168,34 +169,34 @@ This only needs to be run when the system indicates "There is a pending update o
...
+**3. Server Cluster**: Servers in a cluster share all user location registrations. This is the best solution for scaling to thousands of registrations.
 
 !!! note "You won't be able to route calls to users registered against any servers outside the cluster."
 
 !!! tip "Ping Overhead"
-    Due to design constraints that occur with multiple servers in an array, NAT Keep Alive Pings can be quite intensive. You could use *Disable UAC Ping* from the server page, upgrade to a cluster, or use a single registration server to solve this.
+    Due to design constraints that occur with various servers in an array, NAT Keep Alive Pings can be quite intensive. You could use *Disable UAC Ping* from the server page, upgrade to a cluster, or use a single registration server to solve this.
```
Comment (digipigeon) [inline docs/setup/settings/servers.md:199]: Various is not a suitable replacement for multiple in this sentence.

**Review by digipigeon — CHANGES_REQUESTED** [review]

File: docs/setup/settings/servers.md (hunk 1 above)
Comment (Bani2791) [reply docs/setup/settings/servers.md:193]: Replaced **various** with **multiple**

File: docs/setup/settings/servers.md (hunk 2 above)
Comment (Bani2791) [reply docs/setup/settings/servers.md:199]: Replaced **various** with **multiple**

**Comment by Bani2791** [comment]
## **1. Logging**

**Added SIP trace image**

https://bani--connexcs-docs.netlify.app/logging/#sip-traces

## **2. Setup > Advanced > Settings**

### **2.1 Added: Customer :material-menu-right: Routing :material-menu-right: Ingress Routing :material-menu-right: Strategy :material-menu-right: Prefix Set**
**Under:** Step 2: Apply Prefix Set

https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/#configure-prefix-set

### **2.2 Tidy up**

https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/

## **3. Setup > Advanced > Fraud**

### **3.1 Rewrite:** Total Calls The duration of the call.
 
 **Rewritten as:** Total Calls - Total number of calls in a day.
 
 https://bani--connexcs-docs.netlify.app/setup/advanced/fraud/#detection-parameters-and-limits
 
 ### **3.2 Tidyup**
 https://bani--connexcs-docs.netlify.app/setup/advanced/fraud/
 
 ## **4. Customer > Ingress Routing**
 
 **Incorporated Vale Changes**
 
 https://bani--connexcs-docs.netlify.app/customer/routing/
 
 ## **5. Logging**
 
  **Incorporated Vale Changes**
  
 https://bani--connexcs-docs.netlify.app/logging/
 
 ## **6. Setup > Integrations > Portal**
 
  **Incorporated Vale Changes**
  
  https://bani--connexcs-docs.netlify.app/setup/integrations/portal/
  
  ## **6. Setup > Settings > Servers**
 
  https://bani--connexcs-docs.netlify.app/setup/settings/servers/

File: docs/setup/advanced/prefix-set.md [hunk 3]
```diff
@@ -21,29 +27,37 @@ You can apply Prefix Sets to the following locations under the Management menu:
...
 **Regular Expression (RegEx)**
-```
+
+```bash
 ^44(1|2)
 ```
 
 **Combination (UK Landline & Canada)**
-```
+
```
Comment (digipigeon) [inline docs/setup/advanced/prefix-set.md:59]: This is not bash script, it should be text

File: docs/setup/advanced/prefix-set.md [hunk 4]
```diff
@@ -21,29 +27,37 @@ You can apply Prefix Sets to the following locations under the Management menu:
...
+```bash
 441
 442
 ```
 
 **Regular Expression (RegEx)**
-```
+
+```bash
```
Comment (digipigeon) [inline docs/setup/advanced/prefix-set.md:54]: should be text

File: docs/setup/advanced/prefix-set.md [hunk 5]
```diff
@@ -21,29 +27,37 @@ You can apply Prefix Sets to the following locations under the Management menu:
...
-```
+## Example Dial Strings for Prefix Sets
+
+Prefix Sets and all Dial String fields accept plain text or regular expressions. List only one prefix per line.
+
+Here are 3 samples using plain-text, regex, or a mix:
+
+**Plain-Text**
+
+```bash
```
Comment (digipigeon) [inline docs/setup/advanced/prefix-set.md:47]: should be text

**Review by digipigeon — CHANGES_REQUESTED** [review]

File: docs/setup/advanced/prefix-set.md (hunk 3 above)
Comment (Bani2791) [reply docs/setup/advanced/prefix-set.md:55]: done

https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/

File: docs/setup/advanced/prefix-set.md (hunk 5 above)
Comment (Bani2791) [reply docs/setup/advanced/prefix-set.md]: done

https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/

File: docs/setup/advanced/prefix-set.md (hunk 4 above)
Comment (Bani2791) [reply docs/setup/advanced/prefix-set.md]: done
https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by syedrizwansrs — APPROVED** [review]

**Review by digipigeon — APPROVED** [review]

**Pull request merged**